QUICKSTART.md

```
@@ -0,0 +1,98 @@
+
+### Mac/Linux
+```bash
+chmod +x start-all.sh
+./start-all.sh
+```
+
+## 📱 アクセス情報
+
+起動完了後、以下のURLにアクセスできます：
+
+- **🌐 フロントエンド**: http://localhost:3004
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)